dockstore-webservice/src/main/java/io/dockstore/webservice/core/User.java

```diff
@@ -299,15 +307,23 @@ public boolean removeEntry(Entry entry) {
     }
 
     public Set<Entry> getStarredEntries() {
-        return starredEntries;
-    }
+        return starredEntries; }
+
+    public Set<Organization> getStarredOrganizations() {
+        return starredOrganizations; }
```

Comment:
Look for codestyle.xml in the root, can be used by IntelliJ and Eclipse

Reply:
I had a lot of problems with checkstyle when I was writing this, but it might be just how my IntelliJ configuration interacted with the annotation of this class @SuppressWarnings("checkstyle:magicnumber")